typings/react-native-keychain.d.ts

```diff
@@ -72,6 +72,10 @@ declare module 'react-native-keychain' {
     securityLevel?: SECURITY_LEVEL;
     storage?: STORAGE_TYPE;
     rules?: SECURITY_RULES;
+    promptInfoTitle: string;
```

Comment:
I don't think it makes sense to change the API to be required. For example one might only use this on iOS. I'd change it so that the current values are default with an option to override.

Reply:
Didn't think about that. I've made biometric prompt title and negative button optional again.